pipeline/07-export.R

```diff
@@ -397,7 +389,8 @@ if (comp_enable) {
   training_data <- read_parquet(paths$input$training$local) %>%
     filter(!ind_pin_is_multicard, !sv_is_outlier) %>%
     group_by(meta_pin) %>%
-    filter(meta_sale_date == max(meta_sale_date))
+    filter(meta_sale_date == max(meta_sale_date)) %>%
+    ungroup()
```

Comment:
I had previously missed this `ungroup()` call; it doesn't make a difference but is helpful during development to silence grouping warnings.